7.1.цикл_for/7.1.9.py

```diff
@@ -0,0 +1,3 @@
+n = int(input())
+for i in range(n + 1):
+    print('*' * (n - i))
```

Comment:
Это ок, но читается сложно. ИМХО лучше так:
```suggestion
for i in range(n):
    print("*" * n)
    n -= 1
```